scIB/integration.py

```diff
@@ -283,6 +287,18 @@ def runBBKNN(adata, batch, hvg=None):
         corrected = bbknn.bbknn(adata, batch_key=batch, neighbors_within_batch=25, copy=True)
     return corrected
 
+
+def runSaucie(adata, batch):
+    saucie = SAUCIE.SAUCIE(adata.X.shape[1])
+    loader = SAUCIE.Loader(adata.X.todense(), labels=adata.obs[batch])
```

Comment:
densifying is necessary? If that is true, it will make things quite a bit slower here unfortunately.

Reply:
Yes, I tested without densifying and it didn't work